src-built-in/components/workspaceManagementMenu/src/stores/workspaceManagementMenuStore.js

```diff
@@ -342,7 +342,6 @@ Actions = {
 	 * Asks the user if they'd like to save their data, then loads the requested workspace.
 	 */
 	switchToWorkspace: function (data) {
-		if (Actions.getIsSwitchingWorkspaces()) return;
```

Comment:
Is there no longer a need to check this?

Reply:
Nope. Now people can be loading a workspace, decide to switch to another. We will queue the request, attempt to close the current stuff down (and cancel any pending load requests), then switch to the next workspace. If multiple are queued before the close finishes, only the last one requested is used.